src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -179,7 +178,11 @@ public boolean isClosed() {
 
     public void close() {
         if (closeClient && !client.isClosed()) {
-            client.close();
+            try {
+                client.close();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
```

Comment:
I would log warn here and not rethrow unchecked exception, especially since closed will never be executed (could be wrapped in finally block).